test/prob/test_fixture_cdf.hpp

```diff
@@ -533,9 +539,10 @@ class AgradCdfTestFixture : public ::testing::Test {
       add_vars(s2, p0_, p1_, p2_, p3_, p4_, p5_);
       add_vars(s3, p0_, p1_, p2_, p3_, p4_, p5_);
 
-      T_return_type cdf
-          = TestClass.template cdf<Scalar0, Scalar1, Scalar2, Scalar3, Scalar4,
-                                   Scalar5>(p0_, p1_, p2_, p3_, p4_, p5_);
+      T_return_type cdf = pow(
+          TestClass.template cdf<Scalar0, Scalar1, Scalar2, Scalar3, Scalar4,
+                                 Scalar5>(p0_, p1_, p2_, p3_, p4_, p5_),
+          N_REPEAT);
```

Comment:
And this one.

Reply:
Yeah yeah, this is connected to https://github.com/stan-dev/math/pull/2042/files#r483523418 and https://github.com/stan-dev/math/pull/2042/files#r483523944.

This is comparing the results of a vectorized cdf to the equivalent collection of individual cdf calls. It compares something like:

```
cdf([a, a, a, a])
```

To:
```
cdf(a) * cdf(a) * cdf(a) * cdf(a)
```

So the second thing is equivalent to `pow(cdf(a), N)` where `N` is the length of the vector of `a` values in the vectorized call.

What we need to do is verify that `cdf([a, a, a, a])` computes the same thing as `pow(cdf(a), N)`, which is what happens now.

Previously the pow was done separately and so the gradients tried to handle this with some chain rule stuff. I think this was just wrong and wasn't gonna work with higher order stuff.